src/main/java/graphql/execution/instrumentation/ChainedInstrumentation.java

```diff
@@ -71,142 +72,230 @@ public InstrumentationState createState(InstrumentationCreateStateParameters par
     }
 
     @Override
-    public InstrumentationContext<ExecutionResult> beginExecution(final InstrumentationExecutionParameters parameters) {
-        return new ChainedInstrumentationContext<>(map(instrumentations, instrumentation -> {
-            InstrumentationState state = getState(instrumentation, parameters.getInstrumentationState());
-            return instrumentation.beginExecution(parameters.withNewState(state));
+    public InstrumentationContext<ExecutionResult> beginExecution(InstrumentationExecutionParameters parameters) {
+        // these assert methods have been left in so that we truly never call these methods, either in production nor in tests
+        // later when the deprecated methods are removed, this will disappear.
+        return Assert.assertShouldNeverHappen("The deprecated " + "beginExecution" + " was called");
+    }
```

Comment:
FYI this is a breaking change for eg, a downstream ExecutionStrategy implementation that calls `beginExecution(parameters)` and needs to be changed to call `beginExecution(parameters, parametersInstrumentationState)`.  May want to update the 19.0 release notes which claim no breaking changes.

Reply:
thanks a lot @josephlbarnett ... will update the release notes